Document Aspire skills (#1137)
microsoft/aspire.dev: src/frontend/src/content/docs/get-started/ai-coding-agents.mdx

---
title: Use AI coding agents
seoTitle: Use AI coding agents with Aspire AppHost projects today
description: Set up AI coding agents to work with Aspire — install skills, add optional tools, and guide agents through distributed-app workflows.
---

import { Steps, Tabs, TabItem } from '@astrojs/starlight/components';
import AsciinemaPlayer from '@components/AsciinemaPlayer.astro';
import LoopingVideo from '@components/LoopingVideo.astro';

## Why Aspire works well for coding agents

> Distributed apps are hard for agents when the app shape is spread across projects, ports, processes, and terminal output. Aspire turns that into one AppHost model and one set of CLI workflows. Agents can read the graph, start the app, wait for resources to become healthy, inspect failures, and make AppHost changes with compiler feedback.

Reviewer: ```suggestion
Distributed apps are hard for agents when the app shape is spread across projects, ports, processes, and terminal output. Aspire turns that into one AppHost model and one set of CLI workflows. Agents can read resource graphs, start apps, wait for resources to become healthy, inspect failures, and make AppHost changes with compiler feedback.
```

Author: Applied this wording so the intro uses plural resource graphs/apps and reads less tied to one app instance.

> Start with Aspire skills. They give coding agents Aspire-specific guidance for everyday tasks such as app startup, AppHost wiring, debugging, and deployment. If an agent also needs live data from a running app, Aspire includes an optional <abbr title="Model Context Protocol" data-tooltip-placement="top">MCP</abbr> server for resource status, logs, traces, and resource commands.

Reviewer: ```suggestion
The Aspire skills give coding agents Aspire-specific guidance for everyday tasks such as app startup, AppHost wiring, debugging, and deployment. If an agent needs live data from a running app, Aspire includes an optional <abbr title="Model Context Protocol" data-tooltip-placement="top">MCP</abbr> server for resource status, logs, traces, and resource commands.
```

Author: Applied this wording so Aspire skills are the primary guidance and MCP stays positioned as optional live app data.

  cols={256}
  rows={10} />

## Configuration details

> Keep this page focused on setup. Use the dedicated pages when you need the details behind each option:

Reviewer: ???

Author: LOL

Author: Removed this duplicated configuration table from the overview page. The page now goes from setup guidance straight into first prompts, with detailed pages kept in See also.
